actors/runtime/indices/indices.go

```diff
@@ -82,6 +83,8 @@ type IndicesImpl struct {
 	// they are treated as globally available states
 	Epoch                      abi.ChainEpoch
 	NetworkKPI                 big.Int
+	NetworkIndicator           big.Int
+	NetworkTime                big.Int
```

Comment:
Well those are opaque! Please document.

Reply:
will document this in a separate PR, thanks for flagging